handlers/tlsproxy.go

```diff
@@ -9,10 +9,12 @@ import (
 	"strings"
 
 	vhost "github.com/inconshreveable/go-vhost"
+	"github.com/play-with-docker/play-with-docker/services"
 )
 
 func StartTLSProxy(port string) {
 	var validProxyHost = regexp.MustCompile(`^.*pwd([0-9]{1,3}-[0-9]{1,3}-[0-9]{1,3}-[0-9]{1,3})(?:-?([0-9]{1,5}))?\..*$`)
+	var validAliasProxyHost = regexp.MustCompile(`^.*pwd([0-9|a-z|A-Z]*)-([0-9|a-z|A-Z]{8})(?:-?([0-9]{1,5}))?\..*$`)
```

Comment:
hmm we should definitely unify criterias por the regex we use. As aliases could be hostnames, there are other characters that we could use like `-`.